src/Components/WebAssembly/WebAssembly.Authentication/src/Services/AuthorizationMessageHandler.cs

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
+{
+    /// <summary>
+    /// A <see cref="DelegatingHandler"/> that attaches access tokens to outgoing <see cref="HttpResponseMessage"/>s to endpoints
+    /// for requests where any <see cref="Uri"/> in the list of <see cref="Uri"/>s defined in <see cref="AuthorizationMessageHandler.ConfigureHandler(IEnumerable{string}, IEnumerable{string}, string)"/>
+    /// is a base of <see cref="HttpRequestMessage.RequestUri"/>.
```

Comment:
That sentence is a bit long and hard to read :) Suggestion of splitting it into two simplified sentences:

```suggestion
    /// A <see cref="DelegatingHandler"/> that attaches access tokens to outgoing <see cref="HttpResponseMessage"/> instances.
    /// Access tokens will only be added when the request URI is within one of the base addresses configured using
    /// <see cref="AuthorizationMessageHandler.ConfigureHandler(IEnumerable{string}, IEnumerable{string}, string)"/>.
```

Reply:
done